CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(QGeoView LANGUAGES C CXX)
 
 option(BUILD_EXAMPLES "Build examples" ON)
 
+find_package(GDAL CONFIG)
```

Comment:
Is any way to add GDAL also to qmake project?

Reply:
Maybe example from here: https://github.com/chris5287/QWidgetMap/blob/master/src/Example/Example.pro

Reply:
Fine - I will check then myself